src/Combinatorics/Matroids/matroid_strata_grassmannian.jl

```diff
@@ -0,0 +1,417 @@
+export
+    matroid_stratum_matrix_coordinates, matroid_realization_space
+
+@doc Markdown.doc"""
+    matroid_stratum_matrix_coordinates(M::Matroid, B::GroundsetType, F::AbstractAlgebra.Ring = ZZ)
+
+Return the data of the coordinate ring of the matroid stratum of M in the Grassmannian with respect to matrix coordinates. Here, `B` is a basis of `M`` and the submatrix with columns indexed by `B' is the identity. This function returns a pair `(A, W)` where `A` is the coordinate matrix, and `W` is the coordinate ring of the stratum, in general this is a localized quotient ring. 
+
+# Examples
+```jldoctest
+julia> M = fano_matroid();
+julia> (A, W) = matroid_stratum_matrix_coordinates(M, [1,2,4], GF(2));
+julia> A # The coordinate matrix with entries in the polynomial ring `R`.
+[1   0   x[1, 1]   0   x[1, 2]         0   x[1, 4]]
+[0   1   x[2, 1]   0         0   x[2, 3]   x[2, 4]]
+[0   0         0   1   x[3, 2]   x[3, 3]   x[3, 4]]
+julia> W # The coordinate ring of the stratum; in general a localized quotient ring `(R/I)[S⁻¹]`.
```

Comment:
```suggestion
julia> M = fano_matroid();

julia> (A, W) = matroid_stratum_matrix_coordinates(M, [1,2,4], GF(2));

julia> A # The coordinate matrix with entries in the polynomial ring `R`.
[1   0   x[1, 1]   0   x[1, 2]         0   x[1, 4]]
[0   1   x[2, 1]   0         0   x[2, 3]   x[2, 4]]
[0   0         0   1   x[3, 2]   x[3, 3]   x[3, 4]]

julia> W # The coordinate ring of the stratum; in general a localized quotient ring `(R/I)[S⁻¹]`.
```
I think Documenter needs empty lines between these. This might help with the weird `ERROR: UndefVarError: X not defined` in the doctests error.